dev/VSIX/ProjectTemplates/Desktop/CppWinRT/PackagedApp/BlankApp/pch.h

```diff
@@ -21,5 +21,4 @@
 #include <winrt/Microsoft.UI.Xaml.Media.h>
 #include <winrt/Microsoft.UI.Xaml.Navigation.h>
 #include <winrt/Microsoft.UI.Xaml.Shapes.h>
-#include <winrt/Microsoft.UI.Dispatching.h>
-#include <winrt/microsoft.ui.dispatching.co_await.h>
```

Comment:
Consider commenting this out and leaving in with a comment as to where to get it, so there's a breadcrumb for folks that want to add this support.

Reply:
I dropped the relnote on async support into the `readme.txt` that is included with newly created C++/WinRT projects.